app/src/main/res/layout/spend_detail_layout.xml

```diff
@@ -45,84 +46,96 @@
             android:scaleType="center"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
-            app:srcCompat="@drawable/close_white"/>
+            app:srcCompat="@drawable/close_white" />
 
         <ImageView
             android:id="@+id/provider_image"
             android:layout_width="52dp"
             android:layout_height="52dp"
-            android:layout_marginTop="16dp"
             android:layout_marginStart="16dp"
+            android:layout_marginTop="16dp"
             android:background="@drawable/place_holder_rounded_rec_grey"
             android:scaleType="centerCrop"
             app:imageUrl="@{model.providerImageUrl}"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/header_image"/>
+            app:layout_constraintTop_toBottomOf="@+id/header_image" />
+
         <TextView
             android:id="@+id/title"
-            android:textStyle="normal"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_marginTop="16dp"
-            android:layout_marginStart="16dp"
             android:layout_marginEnd="16dp"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="16dp"
             android:ellipsize="end"
             android:fontFamily="sans-serif-medium"
             android:letterSpacing="-0.02"
             android:lines="1"
             android:text="@{model.title}"
             android:textColor="#5c6786"
             android:textSize="22sp"
+            android:textStyle="normal"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="@+id/guideline"
             app:layout_constraintTop_toBottomOf="@+id/header_image"
-            tools:text="$5 Amazon Gift Card"/>
+            tools:text="$5 Amazon Gift Card" />
+
         <android.support.constraint.Guideline
             android:id="@+id/guideline"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:orientation="vertical"
-            app:layout_constraintGuide_begin="68dp"/>
-        <TextView
-            android:id="@+id/info"
-            android:textStyle="normal"
+            app:layout_constraintGuide_begin="68dp" />
+
+        <ScrollView
```

Comment:
Instead of wrapping with ScrollView you could add 
android:scrollbars = "vertical" in the TextView xml definition and 
in call textView.setMovementMethod(new ScrollingMovementMethod()) from kotlin/java code